duckduckgo/Android
# Add vector drawables to use support library

Task/Issue URL: https://app.asana.com/0/1125189844152671/1200809710698334
Tech Design URL: 
CC: 

**Description**:
This PR adds the vector drawables support library into all our internal android library modules

**Steps to test this PR**:
1. Use a device with API 21 and 22
1. Install the app and add a bookmark
1. Go to bookmarks
1. The app should not crash
1. Add a site to unprotected sites
1. Go to settings and unprotected sites
1. The app should not crash
1. Fireproof a site
1. Go to settings and fireproof sites
1. The app should not crash
1. Go to maps.google.com 
1. Click on your location to trigger the location dialog, select use always
1. Go to settings and location
1. App should not crash

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
